README.md

```diff
@@ -18,39 +19,19 @@
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
+1. git clone git@github.com:Have-Samuel/todoList.git
+2. Cd into to-Do-List repo
+3. Use the live server to view the page
```

Comment:
- [ ] When I run the `live server`, the page looks like this.
![Screenshot from 2022-03-24 00-19-34](https://user-images.githubusercontent.com/22434660/159774227-b7c84e05-672f-465d-8776-3686b9c0f693.png)

Please mention the correct step to run the project.

```
# After entering in to the correct folder 

$ npm i
$ npm start
```

- [ ]  Also fix the `Live Demo` section. Please remove that section for now if you don't have a live link to the project.